fastlane/Fastfile

```diff
@@ -39,13 +42,11 @@ lane :set_up_signing do |_options|
     api_key_path: APPLE_API_KEY_PATH,
     readonly: true
   )
-
-  puts lane_context[SharedValues::MATCH_PROVISIONING_PROFILE_MAPPING]
```

Comment:
By the looks of it, in particular the use of `puts`, this was a debugging line that sneaked in.